### DISCUSSION TITLE
Montenegro on sponsorship regions list?

### DISCUSSION THREAD

**Discussion by aelio** [community/community#127944] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Is it planned any time soon?

**Comment by ebndev** [comment, marked as answer]
Hi @aelio, thanks for being a part of the GitHub Community!

GitHub Sponsors is available wherever Stripe is supported so it is dependent on them.

Please visit our [FAQ](https://github.com/orgs/community/discussions/116811) if you have any other questions about Sponsors in general!
